.github/workflows/validate_packs.yml

```diff
@@ -134,9 +134,9 @@ jobs:
       - name: Install dependencies
         run: cd utils && yarn install
 
-      - name: Validate Logos
+      - name: Validate Icons
         id: validation
-        run: cd utils && yarn validate-logos
+        run: cd utils && yarn validate-icons
```

Comment:
this pr should include the entire script change, or at least the rename for this script command.

if this runs w/o that, it will break / fail.